lib/litani.py

```diff
@@ -67,6 +68,48 @@ class TimeoutExpired(Exception):
     pass
 
 
+@dataclasses.dataclass
```

Comment:
I'm not sure that this should be a dataclass. This would synthesize an `__init__` method, when in fact we're not intending to ever instantiate this class. I think `cache_dir_path` should be treated as a static variable, which you're already doing... so I think dataclass is unnecessary